test/uk/gov/hmrc/payetaxcalculatorfrontend/services/DetailedCalcEngineSpec.scala

```diff
@@ -106,6 +106,36 @@ class DetailedCalcEngineSpec extends FreeSpec with Matchers with PropertyChecks
         }
       }
     }
-  }
```

Comment:
All of the below is already tested I'm not sure what value it brings apart from adding noise and magic numbers